src/scripts/audit-events.js

```diff
@@ -56,6 +60,12 @@ const auditComponents = async (rootDir) => {
   }
 }
 
-if (require.main == module) {
-  auditComponents('./src/components')
+// Check if this module is being run directly
+if (import.meta.url.startsWith('file:')) {
```

Comment:
does this actually check if the file is being run directly? That seems to be the bit below